pkg/operator/ceph/cluster/mgr/mgr.go

```diff
@@ -181,3 +215,43 @@ func (c *Cluster) enablePrometheusModule(clusterName string) error {
 	}
 	return nil
 }
+
+// add a servicemonitor that allows prometheus to scrape from the monitoring endpoint of the cluster
+func (c *Cluster) enableServiceMonitor(service *v1.Service) error {
+	name := service.GetName()
+	namespace := service.GetNamespace()
+	serviceMonitor, err := k8sutil.GetServiceMonitor(serviceMonitorFile)
+	if err != nil {
+		return fmt.Errorf("service monitor could not be enabled. %+v", err)
+	}
+	serviceMonitor.SetName(name)
+	serviceMonitor.SetNamespace(namespace)
+	owner := []metav1.OwnerReference{c.ownerRef}
+	serviceMonitor.SetOwnerReferences(owner)
```

Comment:
nit: `k8sutil.SetOwnerRef()` might be simpler.

Reply:
ack